script/app.js

```diff
@@ -0,0 +1,23 @@
+let addToDoButton = document.querySelector(".task__button-add");
+let toDoContainer = document.querySelector(".todo__container");
+let inputField = document.querySelector(".task__input");
+let createTask = document.querySelector(".button_create");
+let deleteTask = document.querySelector(".task__button-delete");
+let taskCard = document.querySelector(".task__card");
+
+createTask.addEventListener("click", function () {
+    if (!taskCard.style.display || taskCard.style.display === "none") {
+        taskCard.style.display = "block";
+    }
+});
+
+deleteTask.addEventListener("click", function () {
+    taskCard.style.display = "none";
+});
+
+addToDoButton.addEventListener("click", function () {
+    let paragraph = document.createElement("p");
+    paragraph.innerText = inputField.value;
```

Comment:
what would happen if the input value is a script? don't we need to check that? Also if the input value is empty should it create an empty card? what if there are unnecessary spaces?

Reply:
issue  solved